testkit/examples/timestamping/proto/mod.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Module of the rust-protobuf generated files.
+
+// For protobuf generated files.
```

Comment:
This is comment attributed to next two lines - which are attributes that allow some bad style in code generated files.

I think since it is inner doc it is informative enough.